src/sdk/cohort.ts

```diff
@@ -0,0 +1,79 @@
+import { PublicKey } from '@nucypher/nucypher-core';
+
+import { Porter } from '../characters/porter';
+import { ChecksumAddress } from '../types';
+
+export type Ursula = {
+  readonly checksumAddress: ChecksumAddress;
+  readonly uri: string;
+  readonly encryptingKey: PublicKey;
+};
+interface CohortParameters {
+  porterUri: string;
+  threshold: number;
+  shares?: number;
+  include?: ChecksumAddress[];
+  exclude?: ChecksumAddress[];
+}
+
+interface CohortJSON {
+  ursulaAddresses: ChecksumAddress[];
+  threshold: number;
+  porterUri: string;
+}
+
+export class Cohort {
+  constructor(
+    public readonly ursulaAddresses: ChecksumAddress[],
+    public readonly threshold: number,
+    public readonly porterUri: string
+  ) {
+    this.ursulaAddresses = ursulaAddresses;
+    this.threshold = threshold;
+    this.porterUri = porterUri;
+  }
```

Comment:
If you use "field declaration as constructor parameter" syntax, you don't need to manually assign values in the constructor body - They will be automatically assigned. 
```suggestion
  ) {}
```